### PR TITLE
Remove more and more warnings

### PR DESCRIPTION
**Pull request by crobibero**
Part of https://github.com/jellyfin/jellyfin/issues/2149

### CODE REVIEW

File: Emby.Server.Implementations/Library/Validators/StudiosValidator.cs [hunk 1]
```diff
@@ -87,12 +87,15 @@ public async Task Run(IProgress<double> progress, CancellationToken cancellation
 
             foreach (var item in deadEntities)
             {
-                _logger.LogInformation("Deleting dead {2} {0} {1}.", item.Id.ToString("N", CultureInfo.InvariantCulture), item.Name, item.GetType().Name);
+                _logger.LogInformation("Deleting dead {ItemType} {ItemId} {ItemName}", item.Id.ToString("N", CultureInfo.InvariantCulture), item.Name, item.GetType().Name);
```
Comment (Bond-009) [inline Emby.Server.Implementations/Library/Validators/StudiosValidator.cs:90]:
```suggestion
                _logger.LogInformation("Deleting dead {ItemType} {ItemId} {ItemName}", item.GetType().Name, item.Id.ToString("N", CultureInfo.InvariantCulture), item.Name);
```

**Review by Bond-009 — APPROVED** [review]

File: Emby.Server.Implementations/Library/Validators/StudiosValidator.cs [hunk 2]
```diff
@@ -87,12 +87,15 @@ public async Task Run(IProgress<double> progress, CancellationToken cancellation
...
             foreach (var item in deadEntities)
             {
-                _logger.LogInformation("Deleting dead {2} {0} {1}.", item.Id.ToString("N", CultureInfo.InvariantCulture), item.Name, item.GetType().Name);
+                _logger.LogInformation("Deleting dead {ItemType} {ItemId} {ItemName}", item.GetType().Name, item.Id.ToString("N", CultureInfo.InvariantCulture), item.Name);
 
-                _libraryManager.DeleteItem(item, new DeleteOptions
+                _libraryManager.DeleteItem(
+                    item,
+                    new DeleteOptions
                 {
```
Comment (cvium) [inline Emby.Server.Implementations/Library/Validators/StudiosValidator.cs:95]: this indentation doesn't look right

Comment (crobibero) [reply Emby.Server.Implementations/Library/Validators/StudiosValidator.cs]: fixed

File: Emby.Server.Implementations/LiveTv/Listings/SchedulesDirectDtos/LineupDto.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        /// <summary>
+        /// Gets or sets the lineup name.
+        /// </summary>
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the transport.
+        /// </summary>
+        [JsonPropertyName("transport.")]
```
Comment (cvium) [inline Emby.Server.Implementations/LiveTv/Listings/SchedulesDirectDtos/LineupDto.cs:27]: You sure about that?

File: Emby.Server.Implementations/LiveTv/Listings/SchedulesDirectDtos/MapDto.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        /// <summary>
+        /// Gets or sets the uhfvhf.
+        /// </summary>
+        [JsonPropertyName("uhfVhf")]
+        public int UhfVhf { get; set; }
+
+        /// <summary>
+        /// Gets or sets the astc major.
+        /// </summary>
+        [JsonPropertyName("astcMajor")]
```
Comment (cvium) [inline Emby.Server.Implementations/LiveTv/Listings/SchedulesDirectDtos/MapDto.cs:39]: No...

File: Emby.Server.Implementations/LiveTv/Listings/SchedulesDirectDtos/MapDto.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        public int AtscMajor { get; set; }
+
+        /// <summary>
+        /// Gets or sets the astc minor.
+        /// </summary>
+        [JsonPropertyName("astcMinor")]
```
Comment (cvium) [inline Emby.Server.Implementations/LiveTv/Listings/SchedulesDirectDtos/MapDto.cs:45]: .

File: MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs [hunk 6]
```diff
@@ -3304,7 +3306,7 @@ public string GetInputModifier(EncodingJobInfo state, EncodingOptions encodingOp
                 inputModifier += " " + videoDecoder;
 
                 if (!IsCopyCodec(state.OutputVideoCodec)
-                    && (videoDecoder ?? string.Empty).IndexOf("cuvid", StringComparison.OrdinalIgnoreCase) != -1)
+                    && videoDecoder.IndexOf("cuvid", StringComparison.OrdinalIgnoreCase) != -1)
```
Comment (cvium) [inline MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs:3309]:
```suggestion
                    && videoDecoder.Contains("cuvid", StringComparison.OrdinalIgnoreCase))
```

File: MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs [hunk 7]
```diff
@@ -3313,7 +3315,7 @@ public string GetInputModifier(EncodingJobInfo state, EncodingOptions encodingOp
 
                     var (width, height) = GetFixedOutputSize(inputWidth, inputHeight, request.Width, request.Height, request.MaxWidth, request.MaxHeight);
 
-                    if ((videoDecoder ?? string.Empty).IndexOf("cuvid", StringComparison.OrdinalIgnoreCase) != -1
+                    if (videoDecoder.IndexOf("cuvid", StringComparison.OrdinalIgnoreCase) != -1
```
Comment (cvium) [inline MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs:3318]:
```suggestion
                    if (videoDecoder.Contains("cuvid", StringComparison.OrdinalIgnoreCase)
```

**Review by cvium — APPROVED** [review]

**Pull request merged**